aten/src/ATen/native/native_functions.yaml

```diff
@@ -10969,18 +10963,30 @@
 - func: linalg_matrix_norm.str_ord_out(Tensor self, str ord='fro', int[] dim=[-2,-1], bool keepdim=False, *, ScalarType? dtype=None, Tensor(a!) out) -> Tensor(a!)
   python_module: linalg
 
-- func: linalg_svd.U(Tensor self, bool full_matrices=True, *, Tensor(a!) U, Tensor(b!) S, Tensor(c!) Vh) -> (Tensor(a!) U, Tensor(b!) S, Tensor(c!) Vh)
+# This function is exposes the `compute_uv` flag, which is then used to implement `linalg.svd` and
+# `linalg.svdvals` as composite functions that call this one
+- func: _linalg_svd(Tensor A, bool full_matrices=False, bool compute_uv=True) -> (Tensor U, Tensor S, Tensor Vh)
```

Comment:
So we replaced one private function (`_svd_helper`) with another one. Are external projects like functorch expected to extend this private function?

Reply:
I reckon they could just consider `svd` and `svdvals` as separate functions, right?
This function is just necessary to be able to write `svdvals` as a composite kernel. Now, I could simply remove it and implement `svdvals` as a function that's not composite. I'm happy to go with either approach.